compiler/x/codegen/UnaryEvaluator.cpp

```diff
@@ -36,6 +36,36 @@
 
 extern TR::Register *intOrLongClobberEvaluate(TR::Node *node, bool nodeIs64Bit, TR::CodeGenerator *cg);
 
+TR::Register *OMR::X86::TreeEvaluator::unaryVectorArithmeticEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Node *valueNode = node->getChild(0);
+   TR::Register *resultReg = cg->allocateRegister(TR_VRF);
+   TR::Register *valueReg = cg->evaluate(valueNode);
+
+   TR::RegisterDependencyConditions *deps = generateRegisterDependencyConditions(0, 1, cg);
+   deps->addPostCondition(resultReg, TR::RealRegister::NoReg, cg);
+   deps->stopAddingConditions();
```

Comment:
You shouldn't need register dependencies unless you are introducing control flow or asking for a specific register.  Functionally it will work and achieve the same result, but they are unnecessary.

I'm not sure why I didn't catch this on the `vneg` PR, but I think the reg deps should be removed from that sequence as well.  Create a separate PR for that.